internal/cmd/jafar/tlsproxy/tlsproxy.go

```diff
@@ -31,13 +32,18 @@ type CensoringProxy struct {
 // the SNII record of a ClientHello. dnsNetwork and dnsAddress are
 // settings to configure the upstream, non censored DNS.
 func NewCensoringProxy(
-	keywords []string, uncensored Dialer,
+	keywords []string, uncensored Dialer, outboundPort *string,
```

Comment:
```suggestion
	keywords []string, uncensored Dialer, outboundPort string,
```

Consistently with my suggestion above